app/views/shared/_site_sidebar.html.erb

```diff
@@ -10,8 +10,8 @@
 
   <h2 class="nav-heading">See also</h2>
   <ul class="nav sidenav flex-column">
```

Comment:
Can we remove this page entirely?  It seems to be part of a feature flag (https://github.com/sul-dlss/exhibits/blob/6fbae9c200cc152b0f7bca72043fd2dc48c2713c/app/views/shared/_site_sidebar.html.erb#LL1C1-L1C1), but I don't think we're ever going to flip it back.

Reply:
I'm not sure, but I'll create an issue to track it.

Reply:
https://github.com/sul-dlss/exhibits/issues/2323